rewrite/src/main/java/vct/col/rewrite/bip/BipVerificationResults.scala

```diff
@@ -0,0 +1,167 @@
+package vct.col.rewrite.bip
```

Comment:
This is the format for importing a verification report that is not tied to an input AST. Not sure where it should go in the project hierarchy; feels a bit out of place here

Reply:
Not sure. Some of the VerCors-as-a-function concerns live in main, maybe that should be decoupled from ui (arg parsing, error printing and whatnot) and this can then live there. Fine for now